src/components/DpSelect/DpSelect.vue

```diff
@@ -99,7 +100,7 @@ export default {
     placeholder: {
```

Comment:
We do not need the whole prop anymore.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/357/commits/44e95c1ccaeef70ce8bd71820e718d91bd6dea8c